src/components/navigation/header/config/iconItems.ts

```diff
@@ -27,4 +30,15 @@ export const iconItems = ({
           }),
       }
     : null,
+  favorites: isLoggedIn
+    ? {
+        ...favoritesLinkConfig,
+        onClick: () =>
+          trackEvent?.({
+            eventCategory: navigationEventCategory,
+            eventAction: 'open_favorites',
```

Comment:
could we also add this event to the favorites link inside the drawer similar to comparison tool? So that we can check what gets used more